run.sh

```diff
@@ -5,7 +5,7 @@
 echo "Getting initial dependencies..."
 
 system_has() {
-  type "$1" > /dev/null 2>&1
+  type "${PWD##*/}" > /dev/null 2>&1
```

Comment:
This should be `$1` since it is the first parameter of `system_has`, not the first of the script.